x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycleTransition.java

```diff
@@ -78,7 +78,8 @@ public static void validateTransition(IndexMetaData idxMeta, Step.StepKey curren
                 "], currently: [" + realKey + "]");
         }
 
-        if (stepRegistry.stepExists(indexPolicySetting, newStepKey) == false) {
+        // Always allow moving to the terminal step, even if it doesn't exist in the policy
```

Comment:
should we document this as it seems like a behavioural change for the `move-to-step` api?

Reply:
I looked through the documentation and didn't see any actual documentation about the terminal step already, so I'm not sure we want to get into that deep of technical behavior in our docs? What do you think?

Reply:
Ah, sure, let's not mention it then